config.go

```diff
@@ -27,24 +27,23 @@ import (
 //     you can find them in GreptimeCloud service detail page.
 //   - Database is the default database the client will operate on.
 //     But you can change the database in InsertRequest or QueryRequest.
-//   - DialOptions and CallOptions are for gRPC service.
-//     You can specify them or leave them empty.
 type Config struct {
 	Host     string // no scheme or port included. example: 127.0.0.1
 	Port     int    // default: 4001
 	Username string
 	Password string
 	Database string // the default database
 
-	keepaliveInterval time.Duration
-	keepaliveTimeout  time.Duration
+	options *options
```

Comment:
suggestion: maybe `tlsOptions` will be a better name? Can we need to expose `options`, for example, `TLSOptions`?